Makefile

```diff
@@ -0,0 +1,48 @@
+#   Copyright The beehive Authors.
```

Comment:
not sure if this is correct 

also the apache 2 license needs to be included

@qizha , @kevin-wangzefeng ?